docs/FeatureManagement/Samples/TargetingFilter.modified.tests.json

```diff
@@ -0,0 +1,50 @@
+{
+    "12": {
```

Comment:
I'm also thinking why we don't directly use array. BTW, why here the index/key starts at "12"?

Reply:
> So, the results of tests 12 through 19 should be True.

Got it. Probably it's because they are subset of the original test suites and you want to reference changed tests results by indices in description? I think we might be simple as below:
- Use the same full test cases, i.e. 1-19.
- Simply tell that Brittney is included after increasing rollout percentage, without mentioning specific tests cases.

Reply:
the update in f804375 LGTM